### PR TITLE
Prepare release 1.5.1

### PR DESCRIPTION
**Pull request by goncafa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.5.1] - 2018-09-12
+### Changed
+- Change example text on modal by the official text.
```
Comment (leosoto) [inline CHANGELOG.md]: This should be in Spanish

Comment (goncafa) [reply CHANGELOG.md]: I dont know why you can not see it. If you go to Files Changes you could check changes are there.

**Comment by goncafa** [comment]
I dont know why you can not see it. If you go to Files Changes you could check changes are there.

**Comment by leosoto** [comment]
@goncafa You are right, I missed the package.json file. But now I don't see the "compiled" javascript updated either.

**Comment by goncafa** [comment]
@leosoto you right. I missed add them.

**Comment by leosoto** [comment]
@goncafa So I was right, but for the wrong reason 😂

**Pull request merged**